src/main/scala/codacy/codesniffer/docsgen/parsers/DocsParser.scala

```diff
@@ -67,7 +67,10 @@ trait DocsParser {
       val parametersDescription = parametersList
         .map(_.map(param => Parameter.Description(param.name, Parameter.DescriptionText(param.name.value))))
       val descriptionWithinLength =
-        description.description.map(descOpt => Pattern.DescriptionText(descriptionTrim(descOpt.value)))
+        description.description.map(desc => {
+          val descriptionWithoutNewLines = desc.value.replaceAll("\n", " ")
```

Comment:
`replaceAll` expects a regex. `replace` works with text so it's more resilient to bad refactorings.

```suggestion
          val descriptionWithoutNewLines = desc.value.replace("\n", " ")
```